app/javascript/components/lease_ups/LeaseUpApplicationsTable.js

```diff
@@ -86,7 +86,7 @@ const LeaseUpApplicationsTable = ({ listingId, dataSet, onLeaseUpStatusChange, o
     const trClassName = classNames(
       'rt-tr-status',
       statusClassName,
-      { 'is-invalid': rowInfo && isInvalid(rowInfo.original) }
+      { 'is-invalid is-disqualified': rowInfo && isInvalid(rowInfo.original) }
```

Comment:
we might be able to drop the is-invalid piece? or do we need to update pattern lib to not make it greyed out?

Reply:
We either need to update pattern lib or leave is-invalid in for grayed out text.

Reply:
does is-invalid do anything other than grayed-out text? we actually want to get rid of the grayed out text piece

Reply:
only that, will remove it then